app/metrics.yaml

```diff
@@ -132,3 +133,112 @@ tabs:
       - fxr-telemetry@mozilla.com
       - manmartin@mozilla.com
     expires: "2020-05-01"
+
+fxa:
+  sign_in:
+    type: counter
+    description: >
+      Number of users who sign in
+    send_in_pings:
+      - baseline
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/1610
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/1854#issuecomment-546214568
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - manmartin@mozilla.com
+    expires: "2020-05-01"
+
+  sign_in_result:
+    type: boolean
+    description: >
+      Number of users who complete the sign in flow successfully
+    send_in_pings:
+      - baseline
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/1610
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/1854#issuecomment-546214568
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - manmartin@mozilla.com
+    expires: "2020-05-01"
+
+  sign_out:
+    type: counter
+    description: >
+      Number of users who sign out by clicking on the sign out button
+    send_in_pings:
+      - baseline
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/1610
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/1854#issuecomment-546214568
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - manmartin@mozilla.com
+    expires: "2020-05-01"
+
+  bookmarks_sync_status:
+    type: boolean
+    description: >
+      Status of the bookmarks sync engine
+    send_in_pings:
+      - baseline
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/1610
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/1854#issuecomment-546214568
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - manmartin@mozilla.com
+    expires: "2020-05-01"
+
+  history_sync_status:
+    type: boolean
+    description: >
+      Status of the history sync engine
+    send_in_pings:
+      - baseline
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/1610
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/1854#issuecomment-546214568
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - manmartin@mozilla.com
+    expires: "2020-05-01"
+
+  sent_tab:
+    type: boolean
+    description: >
+      Sent tab result
```

Comment:
It is a question not a todo. The description mentions this is the send result, but the original question is `number of users who send a tab from FxR`. I am curious if the result is `false`, should we count it in query?

Reply:
Not sure about this one, I've moved this to two events for `sent_tab_success` and `sent_tab_error`. This should give us the number of users who send tabs, per week and also the success rate.